lib/remove-console-statements.js

```diff
@@ -0,0 +1,39 @@
+const cleanConsoleLogs = (j, root) => {
```

Comment:
Please document functions/methods with jsdoc. For switch, almost everything was an impl of a spec, so didn't need inline docs but here, we need proper docs.